README.md

```diff
@@ -14,3 +14,148 @@ Run `cd dist/near-fastauth-wallet && npm publish` to publish npm package
+  btcNetworkId: 'testnet',
+  contract: 'v2.multichain-mpc.testnet',
+});
+
+console.log('Derived BTC Address:', derivedBTCAddress);
+```
+
+#### Example 3: Getting the path
+
+```typescript
+import canonicalize from 'canonicalize';
+
+const path = canonicalize({
+  chain: 60,
+  domain: 'http://localhost:4200',
+});
+
+console.log('Derivation Path:', path);
+```
+
+## `signAndSendMultiChainTransaction`
+
+### Purpose:
+
+The signAndSendMultiChainTransaction method is designed to handle the signing and sending of transactions across multiple blockchain networks. Currently supports EVM and BTC.
+
+### Parameters:
+
+- `data`: This parameter is an object that conforms to the `SendMultichainMessage` type, which can be either `BTCSendMultichainMessage` or `EvmSendMultichainMessage`.
+
+#### For EVM (`EvmSendMultichainMessage`):
+
+- `chain` (number): The unique identifier for the blockchain. Use 60 for EVM. See: [SLIP-0044 documentation](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) for more details.
+- `domain` (optional, string): The key domain used on the derivation path. See: [NEAR Chain Signatures Documentation](https://docs.near.org/build/chain-abstraction/chain-signatures)
+- `to` (string): The recipient address.
+- `value` (bigint): The value to be transferred in wei
+- `meta` (optional, object): Additional metadata for the transaction.
+- `from` (string): The sender address.
+- `chainId` (bigint): The chainId for the EVM network. See: [Chainlist](https://chainlist.org/).
+- `maxFeePerGas` (optional, bigint): The maximum fee per gas unit
+- `maxPriorityFeePerGas` (optional, bigint): The maximum priority fee per gas unit
+- `gasLimit` (optional, number): The gas limit for the transaction
+
+#### For BTC (`BTCSendMultichainMessage`):
+
+- `chain` (number): The unique identifier for the blockchain. Use 0 for BTC. See: [SLIP-0044 documentation](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) for more details.
+- `domain` (optional, string): The key domain used on the derivation path. See: [NEAR Chain Signatures Documentation](https://docs.near.org/build/chain-abstraction/chain-signatures)
+- `to` (string): The recipient address.
+- `value` (bigint): The value to be transferred in satoshis
+- `meta` (optional, object): Additional metadata for the transaction.
+- `from` (string): The sender address.
+- `network` ('mainnet' | 'testnet'): The Bitcoin network type.
+
+### Return Value:
+
+The method returns a promise that resolves once the multi-chain transaction signing request is completed.
+
+### Code Examples:
+
+#### Example 1: Signing an EVM Transaction
+
+```typescript
+const evmTransactionData = {
+  chain: 60,
+  domain: 'http://localhost:4200',
+  to: '0x4174678c78fEaFd778c1ff319D5D326701449b25',
+  value: 10000000000000000n,
+  from: '0x0e80ec32e58cf38eb69ac9bff0adb2e637dc49f5',
+  chainId: 11155111n,
+};
+
+await fastAuthWallet.signAndSendMultiChainTransaction(evmTransactionData);
```

Comment:
it's currently called `signMultiChainTransaction` even though I prefer this naming. I think it's still early enough to change it without a major release? We still have to coordinate with [near-multichain-demo](https://github.com/near/near-multichain-demo) though

Reply:
Good point, as it isn't announced as mainnet, I believe we can introduce breaking changes without a major release. What do you think? 

Regarding sync with `near-multichain-demo` it's already synced on this PR: https://github.com/near/near-multichain-demo/pull/12